controllers/gce/instances/instances.go

```diff
@@ -130,11 +130,15 @@ func (i *Instances) DeleteInstanceGroup(name string) error {
 	}
 	for _, zone := range zones {
 		if err := i.cloud.DeleteInstanceGroup(name, zone); err != nil {
-			if !utils.IsHTTPErrorCode(err, http.StatusNotFound) {
+			if utils.IsNotFoundError(err) {
+				glog.V(3).Infof("Instance group %v in zone %v did not exist", name, zone)
```

Comment:
Not in this PR, but we should standardize our logs. At some places we use `("instance group %v/%v", zone, name)` and at some places we use `("instance group %v in zone %v", name, zone)`